lib/hammer_cli_katello/activation_key.rb

```diff
@@ -56,41 +56,63 @@ def request_params
         field :id, _("Id")
         field :description, _("Description"), Fields::Field, :hide_blank => true
         field :format_consumed, _("Host Limit")
-        field :auto_attach, _("Auto Attach")
+        field :multi_content_view_environment, _("Multi Content View Environment"), Fields::Boolean
         field :release_version, _("Release Version"), Fields::Field, :hide_blank => true
+        field :content_view_environment_labels, _("Content view environments"), Fields::Field
 
-        from :environment do
-          field :name, _("Lifecycle Environment")
+        collection :organization, _("Organization") do
+          field :id, _("Id"), Fields::Field, :hide_blank => true
+          field :name, _("Name"), Fields::Field, :hide_blank => true
         end
-        from :content_view do
-          field :name, _("Content View")
+
+        collection :content_view_environments, _('Content view environments') do
+          from :content_view do
+            label _("Content view") do
+              field :id, _("Id")
+              field :name, _("Name")
+              field :content_view_version, _("Version")
+              field :content_view_version_id, _("Content view version Id")
+              field :composite, _("Composite"), Fields::Boolean
+              field :content_view_environment_id, _("Content view environment id"), Fields::Field
+            end
+          end
+          from :lifecycle_environment do
+            label _("Lifecycle environment") do
+              field :id, _("Id")
+              field :name, _("Name")
+            end
+          end
+          field :label, _("Environment Label")
```

Comment:
Saying 'Environment Label' here will imply that it's about lifecycle environment.

```suggestion
          field :label, _("Label")
```